src/emulator/extensionsEmulator.ts

```diff
@@ -169,4 +174,25 @@ export class ExtensionsEmulator {
       STORAGE_BUCKET: `${projectId}.appspot.com`,
     };
   }
+
+  private async checkAndWarnAPIs(instances: planner.InstanceSpec[]): Promise<void> {
+    const apisToWarn = await getUnemulatedAPIs(this.args.projectId, instances);
+    if (apisToWarn.length) {
+      const table = new Table({
+        head: ["API Name", "Instances using this API", `Enabled on ${this.args.projectId}`],
+        style: { head: ["yellow"] },
+      });
+      for (const apiToWarn of apisToWarn) {
+        table.push([apiToWarn.apiName, apiToWarn.instanceIds, apiToWarn.enabled]);
```

Comment:
have we considered giving users a way out on the disabled APIs?

If they have an instance that uses translate API, but they don't have that enabled, they are basically stuck. Maybe we can ask if they want to enable the apis or log a url to pantheon?

Reply:
This is a good point. I like the idea of including a link to enable the API in console - I think we want to be _very_ careful that users know that they are taking an action that affects production, & offering a prompt to enable them might not make that abundantly clear.

Reply:
sgtm, that's an excellent point, I'd be one of those people that spam enter through everything lol